tests/crud/test_serializers.py

```diff
@@ -256,3 +256,29 @@ class Computer2(Table):
 
         with self.assertRaises(ValueError):
             create_pydantic_model(Computer, exclude_columns=(Computer2.CPU,))
+
+
+class TestNestedModel(TestCase):
+    def test_nested_models(self):
+        class Country(Table):
+            name = Varchar(length=10)
+
+        class Director(Table):
+            name = Varchar(length=10)
+            country = ForeignKey(Country)
+
+        class Movie(Table):
+            name = Varchar(length=10)
+            director = ForeignKey(Director)
+
+        MovideModel = create_pydantic_model(table=Movie, nested=True)
+        DirectorModel = create_pydantic_model(table=Director, nested=True)
+        CountryModel = create_pydantic_model(table=Country, nested=True)
```

Comment:
I think technically we shouldn't need to pass `nested=True` for this one. But not doing so, will fail the latest assertion.
Even though `Country` doesn't have `ForeignKey` but there's a difference between `CountryModel` created with and without `nested=True` and I guess that's related to the `@lru_cache` hashing.

Reply:
You're right, it's probably the `lru_cache`.